staging/src/k8s.io/apiserver/pkg/storage/cacher/cacher_whitebox_test.go

```diff
@@ -980,7 +981,8 @@ func TestBookmarksOnResourceVersionUpdates(t *testing.T) {
 		for {
 			event, ok := <-w.ResultChan()
 			if !ok {
-				t.Fatalf("Unexpected closed channel")
+				t.Errorf("Unexpected closed channel")
+				continue
```

Comment:
seems like a return is more suitable.  you know this won't work at this point, right?  You don't want to hot loop without a return.